cms/urls.py

```diff
@@ -11,15 +11,19 @@
 from wagtail.documents import urls as wagtaildocs_urls
 from wagtail.utils.urlpatterns import decorate_urlpatterns
 
+from cms.core import views as core_views
 from cms.core.cache import get_default_cache_control_decorator
 
 if TYPE_CHECKING:
     from django.urls import URLPattern, URLResolver
 
+# Internal URLs are not intended for public use.
+internal_urlpatterns = [path("readiness/", core_views.ready, name="readiness")]
 
 # Private URLs are not meant to be cached.
 private_urlpatterns = [
     path("documents/", include(wagtaildocs_urls)),
+    path("-/", include((internal_urlpatterns, "internal"))),
```

Comment:
question: will this restricted to internal IPs only?

Reply:
In future, I'd like it to be, yes, if not further. Having the namespace here ready for those kinds of URLs makes it easier to add them in future. It might be YAGNI, but it's not much extra overhead at the moment.

Reply:
My browser doesn't seem to like this?.
Why not just use `internal`, it's explicit. We need to, at some point, do a mapping to ensure our URLs don't overlap with existing URLs on the site.